Readability.js

```diff
@@ -1598,6 +1598,7 @@ Readability.prototype = {
 
     var tagsList = e.getElementsByTagName(tag);
     var curTagsLength = tagsList.length;
+    var isList = ["ul", "ol"].indexOf(tag) !== -1;
```

Comment:
The idea was to ease adding more tags in the future, but for lists I think we're already covering everything.